### PR TITLE
toolbar update

### PR DESCRIPTION
**Pull request by DMachalek**

### CODE REVIEW

**Review by DMachalek — COMMENTED** [review]
Akhil feedback

**Review by DMachalek — COMMENTED** [review]
second review

File: packages/ove/src/ToolBar/style.css [hunk 1]
```diff
@@ -136,3 +165,19 @@
...
+}
+
+.veToolbarItemOuter.ve-tool-container-findTool
+  .bp3-input-group
+  .bp3-input-action,
+.veToolbarItemOuter.ve-tool-container-findTool
+  .bp3-input-group
+  > .bp3-input-left-container,
+.veToolbarItemOuter.ve-tool-container-findTool .bp3-input-group > .bp3-button,
+.veToolbarItemOuter.ve-tool-container-findTool .bp3-input-group > .bp3-icon {
```
[the remark this answers is not in the document]
Comment (DMachalek) [reply packages/ove/src/ToolBar/style.css:180]: simplification does not fix it

**Review by AkhilECL — APPROVED** [review]
Nice work!

**Pull request merged**